src/ViewRendererBundle/Bridge/Chameleon/Migration/Script/update-1539324183.inc.php

```diff
@@ -27,5 +27,8 @@
 ;
 TCMSLogChange::update(__LINE__, $data);
 
-TCMSLogChange::addInfoMessage('Removing "chameleon/outbox/static/less/cached" from additionally to clean cache directories.
-Is now handled as part of the system - path can be configured.');
+TCMSLogChange::addInfoMessage('The path for generated CSS can now be configured using bundle config
```

Comment:
Why two messages? The second one is a (more or less) technical detail of the first.

Reply:
Exactly because of this - the fact that the path is now configurable shouldn't be hidden as some detail within another message. I think both pieces of information are important and distinct enough to justify separate messages.

Reply:
I meant it the other way around...